app/helpers/forms_helper.rb

```diff
@@ -110,18 +110,6 @@ def radio_with_label(**args)
     end
   end
 
-  def radio_button_with_label(**args)
-    args = auto_label_if_form_is_account_prefs(args)
-    opts = separate_field_options_from_args(args)
-
-    wrap_class = form_group_wrap_class(args, "btn btn-default btn-sm")
-
-    args[:form].label(args[:field], class: wrap_class) do
```

Comment:
@mo-nathan Sorry, you're right.

This is me not cleaning up after myself - I wrote it for the index sort-by buttons and then didn't use it, because it didn't work as intended. Let's delete.